src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -44,7 +44,7 @@ export default class CourtOrderPoa extends Vue {
 
   /** Prompt the validations. Used for global validations. */
   @Prop({ default: false })
-  private validate
```

Comment:
I needed to change the prop's name because it conflicted with FormIF

Reply:
It wouldn't hurt to add a type to this prop.

Also, props should be readonly (not private).

Reply:
Yes, you're right. It should be public. 👍

Reply:
Well, it doesn't need to be public since nothing external needs to use this class variable.

Reply:
Expose or not expose? 🤔

Reply:
My point is it should be readonly.

As it turns out, public/private doesn't work in TS at the moment anyway so 🤷‍♂️

Reply:
ok. Sounds good.